src/components/classrooms/AddUserInClassroomButton.js

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from 'react';
+import AddIcon from '@material-ui/icons/Add';
+import { withStyles } from '@material-ui/core';
+import Fab from '@material-ui/core/Fab';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withTranslation } from 'react-i18next';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { addUserInClassroom } from '../../actions';
+import {
+  ADD_USER_IN_CLASSROOM_DIALOG_TITLE_ID,
+  ADD_USER_IN_CLASSROOM_NAME_INPUT_ID,
+  ADD_USER_IN_CLASSROOM_BUTTON_ID,
+  ADD_USER_IN_CLASSROOM_VALIDATE_BUTTON_ID,
+  ADD_USER_IN_CLASSROOM_CANCEL_BUTTON_ID,
+} from '../../config/selectors';
+import { isUsernameValid } from '../../utils/user';
+
+const styles = theme => ({
+  fab: {
+    position: 'absolute',
```

Comment:
To change to `fixed` in order to always have the icon in the bottom right-hand-side of the screen, even when we change the displayed number of rows.